debian/kolibri-server.init

```diff
@@ -169,5 +169,3 @@ case "$1" in
     exit 3
   ;;
 esac
-
-:
```

Comment:
TBH, I deleted this because I didn't know the meaning of a `:` and thought it was a syntax error. Turns out it is the same as `true` -- but I think it's meaningless here. In other cases above, there is `exit N` which will exit the script and prevent anything appended after it to run. But the `:` needs an exit to terminate anything, by itself, it seems meaningless.